src/main/java/uk/gov/hmcts/reform/locationrefdata/configuration/BatchConfig.java

```diff
@@ -18,7 +17,6 @@
 import uk.gov.hmcts.reform.locationrefdata.camel.task.LrdOrgServiceMappingRouteTask;
 
 @Configuration
-@EnableBatchProcessing
```

Comment:
Keeping this in causes Spring Batch to no longer auto-create its tables in the DB.